horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/TestServiceImpl.java

```diff
@@ -321,7 +321,9 @@ public TestListing summary(String roles, String folder, Integer limit, Integer p
          testSql.append(" WHERE COALESCE(folder, '') = COALESCE((?1)::::text, '')");
          Roles.addRolesSql(identity, "test", testSql, roles, 2, " AND");
       }
-      Util.addPaging(testSql, limit, page, "test.name", direction);
+      if( limit > 0 && page > 0 ) {
```

Comment:
is `page` 0-indexed?

Reply:
Default in the annotation is 1

Reply:
ok, as long as we can just return the first page